servicetalk-grpc-api/src/main/java/io/servicetalk/grpc/api/GrpcClientBuilder.java

```diff
@@ -270,11 +270,25 @@
      * <p>
      * This setting disables the default filter such that no {@code Host} header will be manipulated.
      *
+     * @deprecated Use {@link #hostHeaderFallback(boolean)}.
```

Comment:
note with updated Checkstyle rules the `@deprecated` javadoc tag must be the last tag.